Mathlib/CategoryTheory/Preadditive/Injective.lean

```diff
@@ -358,24 +385,12 @@ theorem map_injective_iff (P : C) : Injective (F.functor.obj P) ↔ Injective P
 /-- Given an equivalence of categories `F`, an injective presentation of `F(X)` induces an
 injective presentation of `X.` -/
 def injectivePresentationOfMapInjectivePresentation (X : C)
-    (I : InjectivePresentation (F.functor.obj X)) : InjectivePresentation X where
-  J := F.inverse.obj I.J
-  injective := Adjunction.map_injective F.toAdjunction I.J I.injective
-  f := F.unit.app _ ≫ F.inverse.map I.f
-  mono := mono_comp _ _
+    (I : InjectivePresentation (F.functor.obj X)) : InjectivePresentation X :=
+  F.toAdjunction.injectivePresentationOfMapInjectivePresentation _ I
 #align category_theory.equivalence.injective_presentation_of_map_injective_presentation CategoryTheory.Equivalence.injectivePresentationOfMapInjectivePresentation
 
-theorem enoughInjectives_iff (F : C ≌ D) : EnoughInjectives C ↔ EnoughInjectives D := by
-  constructor
-  all_goals intro H; constructor; intro X; constructor
-  · exact
-      F.symm.injectivePresentationOfMapInjectivePresentation _
-        (Nonempty.some (H.presentation (F.inverse.obj X)))
-  · exact
-      F.injectivePresentationOfMapInjectivePresentation X
-        (Nonempty.some (H.presentation (F.functor.obj X)))
+theorem enoughInjectives_iff (F : C ≌ D) : EnoughInjectives C ↔ EnoughInjectives D :=
+  ⟨fun h => h.of_adjunction F.symm.toAdjunction, fun h => h.of_adjunction F.toAdjunction⟩
 #align category_theory.equivalence.enough_injectives_iff CategoryTheory.Equivalence.enoughInjectives_iff
 
 end Equivalence
-
-end CategoryTheory
```

Comment:
It's a bit strange that Lean now doesn't require every namespace/section be ended ...